diff --git a/decision_transformer/generate_dataset.py b/decision_transformer/generate_dataset.py
index ff23d33..6ab7c65 100644
--- a/decision_transformer/generate_dataset.py
+++ b/decision_transformer/generate_dataset.py
@@ -11,7 +11,7 @@ import wandb
 from wandb.integration.sb3 import WandbCallback
 
 import gym
-import vizdoomgym.gym_wrapper
+from vizdoom import gym_wrapper
 
 config = {
         "policy_type": 'CnnPolicy',
@@ -34,7 +34,7 @@ def parse_args():
 
 
 def make_env():
-    env = make_vec_env(config['env_name'], n_envs=config['num_cpu'], seed=0, vec_env_cls=SubprocVecEnv)
+    env = gym.make(config["env_name"])
     env = Monitor(env)
     env = WarpFrame(env)
     return env
diff --git a/decision_transformer/wandb/latest-run b/decision_transformer/wandb/latest-run
index 822fba3..8d015cb 120000
--- a/decision_transformer/wandb/latest-run
+++ b/decision_transformer/wandb/latest-run
@@ -1 +1 @@
-run-20220919_062206-4dpinmno
\ No newline at end of file
+run-20230130_120658-2jditkax
\ No newline at end of file
